kubernetes/fineract-server-deployment.yml

```diff
@@ -61,16 +61,23 @@ spec:
       containers:
       - name: fineract-server
         image: apache/fineract:latest
+        resources:
+          limits:
+            cpu: "1000m"
+            memory: "1Gi"
+          requests:
+            cpu: "200m"
+            memory: "0.5Gi"
         livenessProbe:
           httpGet:
             path: /fineract-provider/actuator/health/liveness
-            port: 8443
+            port: 8080
```

Comment:
I feared that this may not work (because we have force http to https redirection), so I've briefly tested it, and to my surprised found that on the /actuator/ URLs we don't have that, so http://localhost:8080/fineract-provider/actuator/health/liveness does work! All good then.  (https://localhost:8443/fineract-provider/actuator/health/liveness also works, IFF you use e.g. `http --verify=no get`, I suspect this is to avoid issues with the self signed default certificate).